catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/api/FlatFilterBuilder.java

```diff
@@ -53,10 +54,16 @@
 
   FlatFilterBuilder beginNilType();
 
+  FlatFilterBuilder addFunctionType(String functionName, List<Object> args);
```

Comment:
❓  is there a reason why some of these are prefixed with `begin..` , `set..`, vs `add`? are these all functionally different operations?